src/less/variables.less

```diff
@@ -33,6 +33,8 @@
 @global-color-warning: #f2c746;
 @global-color-warning-light: tint(@global-color-warning, 92%);
 @global-color-emphasis: #fdf9c7;
+@global-color-green-lightest: #dbedb2;
```

Comment:
Let's try to follow our current pattern here. 

The green color is a `success` color, so it should go underneath `@global-color-success-light`

Red is `danger` and should go underneath `@global-color-danger-light`.

If we can, let's also use the tint function like in the other cases so that all color variations are not standalone, but instead **derived** from our base theme colors. If this is too difficult to match then we can disregard but I think we should be able to get really close to our target. That way if product decides to modify our theme colors in any way, the corresponding shades will updated as well.

Reply:
@juanfabrega The color family is actually very different from the existing base green and red colors we have, so adding tint is not working out in this case. Hence, I am updating the variable names as shared by you.

Reply:
Sounds good, and makes sense!